packages/react-native/sdks/hermes-engine/hermes-utils.rb

```diff
@@ -81,6 +81,10 @@ def hermes_engine_tarball_envvar_defined()
     return ENV.has_key?('HERMES_ENGINE_TARBALL_PATH')
 end
 
+def react_native_maven_namespace()
+    return 'com.facebook.react'
```

Comment:
I would inline this in the method below, given that it is used only once (also, as it is, it is fail. This name space is `.` separated, but in the url it must be `/` separated)

Reply:
Fixed